### PR TITLE
test browser for replace manuscript

### LINKED ISSUE #1319: create test browser to ensure replace manuscript is working

### PR DESCRIPTION
**Pull request by jyenes**
test browser for replacing manuscript, and fix the issue avoiding replace manuscript

#### Background

What does this PR do?
closes #1319 

Please indicate the need for this change to be tested. If not please justify, if so then at what levels. 

- [x] Browser tests

### CODE REVIEW

File: server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js [hunk 1]
```diff
@@ -133,10 +133,6 @@ async function uploadManuscript(_, { file, id, fileSize }, { user }) {
     manuscript.files[fileIndex] = fileEntity
   }
 
-  // change the file entity from pending
-  fileEntity.type = 'MANUSCRIPT_SOURCE'
-  fileEntity.save()
```
Comment (Cooryd) [inline server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js]: Is this a fix for the double uploading of a `MANUSCRIPT_SOURCE` or is this just reverting it back to how it was?

File: server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ async function uploadManuscript(_, { file, id, fileSize }, { user }) {
     manuscriptId: id,
     url: `manuscripts/${id}`,
     filename,
-    type: 'MANUSCRIPT_SOURCE_PENDING',
+    type: 'MANUSCRIPT_SOURCE',
```
Comment (diversemix) [inline server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js]: @jyenes Can you remove the changes to this file as its going to hide the existing problem

**Review by diversemix — CHANGES_REQUESTED** [review]
Tests look good

**Review by diversemix — APPROVED** [review]

**Comment by jyenes** [comment]
this should work, as soon as the issue on replacing manuscript is fixed

**Pull request merged**